package.json

```diff
@@ -75,7 +76,7 @@
     "identity-obj-proxy": "^3.0.0",
     "jest": "^26.6.3",
     "jest-styled-components": "^7.0.3",
-    "lerna": "^5.3.0",
+    "lerna": "^5.4.0",
```

Comment:
I'm confused. I just set lerna to `^5.4.0` in main. Why is it showing this diff?